theme.js

```diff
@@ -978,6 +978,13 @@ export default () => ({
       },
     },
 
+  '.inline': {
+    container: {
+      width: window.width,
+        position: 'relative',
```

Comment:
Awesome! Sorry for the indentation =(

Yeah I tested on `ListView`.. going to take a look on `ScrollView` component.